templates/admin-page.php

```diff
@@ -31,7 +31,7 @@
 					<?php } ?>
 				</select>
 
-				<input type="submit" value="<?php esc_attr_e( 'Check it!', 'plugin-check' ); ?>" />
+				<input type="button" value="<?php esc_attr_e( 'Check it!', 'plugin-check' ); ?>" id="pc_check_it" />
```

Comment:
Rather than changing the submit type, could you call `event.preventDefault()` in your event listener? I think it's also better to listen for the submit event for the whole form, rather than a click event on this button, which wouldn't be triggered by a keyboard event, like someone hitting "enter" while focused on the form.

Reply:
Updated it now.